modules/cluster-logging-elasticsearch-retention.adoc

```diff
@@ -0,0 +1,120 @@
+// Module included in the following assemblies:
+//
+// * logging/cluster-logging-elasticsearch.adoc
+
+[id="cluster-logging-elasticsearch-retention_{context}"]
+= Configuring log retention time
+
+You can specify how long the default Elasticsearch log store keeps indices 
+using a separate _retention policy_ for each of the three log sources: 
+infrastructure logs, application logs, and audit logs. The retention policy, 
+configured using the `maxAge` parameter in the Cluster Logging Custom Resource 
+(CR), determines when to delete the Elasticsearch indices. 
+
+Elasicsearch rolls over an index, moving the current index and creating a new 
+index, when an index matches any of the following conditions:
+
+* the index is older than than the `rollover.maxAge`.
+* the index size is greater than 40gb*number of primary shards.
+* the index doc count is greater than 40960*number of primary shards.
+
+If you do not create a retention policy for any of the log sources, logs 
+are retained for seven days by default. If you do not specify a retention policy 
+for all three log sources, logs from only the sources with a retention policy 
+are stored. For example, if you do not set a retention policy for audit logs, 
+the audit logs will not be retained and there will be no *audit-* index in 
```

Comment:
@mburke5678 Per my understanding, the audit logs won't be collected by default, even though there has `audit-*` index in the ES.
Besides, I'm not sure if this behavior(If you do not specify a retention policy 
for all three log sources, logs from only the sources with a retention policy 
are stored.) is expected or not, let's wait for Jeff's reply.